### PR TITLE
MYMATH-1 Implement prime checker

### PR DESCRIPTION
**Pull request by faragocsaba**

### CODE REVIEW

File: is_prime.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def is_prime(n):
+    if n <= 1:
+        return False
+    for i in range(2, int(n ** 0.5) + 1):
```
Comment (farago-csaba-webuni) [inline is_prime.py:4]: Optimize the cycle so that it runs until the square root of n.

File: is_prime.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        if n % i == 0:
+            return False
+    return True
+
+
+if __name__ == '__main__':
+    for i in range(1, 20):
```
Comment (farago-csaba-webuni) [inline is_prime.py]: if __name__ == '__main__'.

**Review by farago-csaba-webuni — CHANGES_REQUESTED** [review]

File: is_prime.py (hunk 1 above)
Comment (faragocsaba) [reply is_prime.py:4]: Changed.

File: is_prime.py (hunk 2 above)
Comment (faragocsaba) [reply is_prime.py]: Added.

File: is_prime.py (hunk 1 above)
Comment (farago-csaba-webuni) [reply is_prime.py:4]: The range end should be int(n ** 0.5) + 1.

Comment (faragocsaba) [reply is_prime.py:4]: Good catch! Fixed.

**Review by farago-csaba-webuni — APPROVED** [review]

**Pull request merged**